trident/utilities.py

```diff
@@ -210,7 +210,7 @@ def get_datafiles(datadir=None, url=None):
     >>> get_datafiles()
     """
     if datadir is None:
-        datadir = expanduser('~/.trident')
+        datadir = expanduser(os.path.join('~','.trident'))
```

Comment:
Might as well clean up the imports while you're here as well. I suggest removing the `from os.path import expanduser` and use `os.path.expanduser` here.

Reply:
Done.